vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/test/java/com/vaadin/flow/component/combobox/dataview/ComboBoxDataViewTest.java

```diff
@@ -225,10 +222,6 @@ public void setSortComparator(
         // eagerly with in-memory data provider
         comboBox.setOpened(true);
 
-        // We expect that the current implementation of 'setItems' with IMDP
-        // will delegate to 'setItems(DataProvider)'
-        Mockito.verify(comboBox).setItems(dataProviderCaptor.capture());
```

Comment:
Similar problem here where the test setup relied on one `setItems` method calling another one, in order to access the data provider. Used same solution as above, just access the data provider directly after setting items.